data.sql

```diff
@@ -1,21 +1,190 @@
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Agumon', '02/03/2020', 0, TRUE, 10.23);
 
-INSERT INTO animals
-(name, date_of_birth, escape_attempts, neutered, weight_kg) 
-VALUES ('Agumon', '3-2-2020', '0', 'true', '10.23');
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Gabumon', '11/15/2018', 2, TRUE, 8);
 
-INSERT INTO animals
-(name, date_of_birth, escape_attempts, neutered, weight_Kg)
-VALUES ('Gabumon', '11-15-2018', '2', "true", '8')
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Pikachu', '01/07/2011', 1, FALSE, 15.04);
 
-INSERT INTO animals
-(name, date_of_birth, escape_attempts, neutered, weight_Kg)
-VALUES ('Pikachu', '7-1-2021', '1', 'false', '15.04')
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Devimon', '05/12/2017', 5, TRUE, -11);
 
-INSERT INTO animals
-(name, date_of_birth, escape_attempts, neutered, weight_Kg)
-VALUES ('Devimon', '12-5-2017', '5', 'true', '11')
\ No newline at end of file
+-- Her name is Plantmon. She was born on Nov 15th, 2022, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
+
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Charmander', '08/02/2020', 0, FALSE, -11);
+
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Plantmon', '15/11/2022', 2, TRUE, -5.7);
+
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Squirtle', '02/04/1993', 3, FALSE, -12.13);
+
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Angemon', '12/06/2005', 1, TRUE, -45);
+
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Boarmon', '07/06/2005', 7, TRUE, 20.4);
+
+INSERT INTO animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Blossom', '10/13/1998', 3, TRUE, 17);
+
+-- for owmers table
+
+INSERT INTO owners (
+  full_name,
+  age
+)
+VALUES ('Sam Smith ', 34)
+
+INSERT INTO owners (
+  full_name,
+  age
+)
+VALUES ('Jennifer Orwell', 19)
+
+INSERT INTO owners (
+  full_name,
+  age
+)
+VALUES ('Melody Pond', 77)
+
+INSERT INTO owners (
+  full_name,
+  age
+)
+VALUES ('Dean Winchester', 14)
+
+INSERT INTO owners (
+  full_name,
+  age
+)
+VALUES ('Jodie Whittaker', 38)
+
+-- species
+
+INSERT INTO species (
+  name
+)
+VALUE ('Pokemon')
+
+INSERT INTO species (
+  name
+)
+VALUE ('Digimon')
+
+BEGIN;
+UPDATE animals SET species_id = 'Digimon' WHERE name LIKE '%mon';
```

Comment:
Please select the `id` from `species` where the name is `Digimon` and assign it to `species_id` whereas currently, you are assigning `Digimon` as `species_id`.